src/components/page-types/searchPage.js

```diff
@@ -156,7 +156,7 @@ const SearchPage = (props) => {
         <Container
           element="section"
           width="site"
```

Comment:
```suggestion
          width="full"
```
@Lyafel There's an easier way for this too 😄  You could then remove the classname below. Please double check though.

Reply:
@yvonnetangsu Good to know again :D though site adds su-cc, the paddings, and full doesn't so regardless of the attribute, a class will have to be added, either su-cc for full or width for site?